apps/publications/src/app/pages/publication-detail/publication-detail.component.ts

```diff
@@ -93,20 +100,25 @@ export class PublicationDetailComponent implements OnInit {
     this.cabinetService.findPublicationById(this.publicationId).subscribe(publication => {
       this.publication = publication;
       this.thanks = this.publication.thanks;
+      this.disabledColumns = publication.locked ? ['id', 'name', 'organization', 'email'] : ['select', 'id', 'name', 'organization', 'email'];
```

Comment:
When I change the publication from unlocked to locked the select checkbox doesnt hide as it should 
![image](https://user-images.githubusercontent.com/32871167/119062573-6778a880-b9d7-11eb-8659-943d25ce93e3.png)